module/network/RobotCommunication/data/config/RobotCommunication.yaml

```
@@ -1,5 +1,5 @@
 # Controls the minimum log level that NUClear log will display
-log_level: INFO
+log_level: DEBUG
```

when you've finished testing change back to INFO

```suggestion
log_level: INFO
```